### PR TITLE
Various reliability fixes

### PR DESCRIPTION
**Pull request by bcook-r7**
This incorporates some fixes from @Sliim and bumps us to Virtualbox 5.1.10

### CODE REVIEW

**Comment by Sliim** [comment]
Cool! :+1: 
Sorry, too busy to send a PR, thanks to have reported my changes here! :)
Note: I have still an issue with the administrator user, after provisioning, the auto-login doesn't work because it tries to connect as "vagrant" user name, that was renamed to "Administrator"..

**Comment by bcook-r7** [comment]
Thanks @Sliim , are you using a different base box? We could probably rename our base user in packer to Administrator for consistency, since I think that's probably less surprising.

**Comment by Sliim** [comment]
Yes, I'm using one of this box: https://atlas.hashicorp.com/opentable
I'm currently testing the metasploitable3 with this box: https://atlas.hashicorp.com/opentable/boxes/win-2008r2-standard-amd64-nocm. 
Note that with these box, I got some issues related to https://github.com/rapid7/metasploitable3/issues/4. I fixed my base box by increasing `@{MaxConcurrentOperationsPerUser="200"}` and cie, and then repackage the box.

These box comes with `vagrant` user as default admin too, like many base box for vagrant, so it's not weird to have this user created with packer. Maybe we can change the auto-logon configuration with a provisioning script?

File: scripts/installs/openssh.ps1 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-param (
+aram (
```
Comment (Sliim) [inline scripts/installs/openssh.ps1]: Typo here

**Comment by busterb** [comment]
I guess these are superceded.

**Pull request closed without merging**